content/privacy.md

```diff
@@ -8,12 +8,19 @@ type: page
 ## Who we are
 
 ## Your personal information
+
 Your personal information belongs to you. Any personal information we get from you will only be used internally and never shared with third parties.
 
+### Contact information
+
+> If we ask for your email address we will only use it to contact you regarding sessions you signed up for, or to notify of future sessions.
+
 ## Public code reading sessions
 
 ### Sharing tools
 
+In our remote public code reading sessions we use collaborative tools to allow us to talk to each other and to collect discussion points from attendees.
+
```

Comment:
> I need help with this wording: 

In order to keep it simple, many of these resources are accessible on links which allow universal access. We will aim to make these private or remove them from the internet as soon as possible after a session, but sometimes, the artefacts from a session might remain online for a period of time.

Reply:
I'm not sure what you mean by artefacts so maybe some examples would be good here.

Other than that, what do you think about these changes:

To keep it simple, we may use tools that are accessible using public links. After each session we will aim to make these private or remove them from the internet, but sometimes artefacts from a session might remain online for a period of time.

Reply:
Thanks - I tweaked and used that.